admin/resources/js/api/poolOperations.graphql

```diff
@@ -9,9 +9,10 @@ query getPools {
       en
       fr
     }
-    description {
```

Comment:
Any reason to remove the description field as it still seems to be part of the [pool model](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/api/app/Models/Pool.php#L35)?

Reply:
Just don't need it, and this seemed to be the only place the query is called. No reason to return extra data we don't need, was my thought. But I added it back in any who